source/ui/BioXAS/BioXASSSRLMonochromatorRegionControlEditor.cpp

```diff
@@ -2,7 +2,7 @@
 #include "beamline/BioXAS/BioXASSSRLMonochromatorRegionControl.h"
 
 BioXASSSRLMonochromatorRegionControlEditor::BioXASSSRLMonochromatorRegionControlEditor(BioXASSSRLMonochromatorRegionControl *regionControl, QWidget *parent) :
-	BioXASControlEditor(regionControl, parent)
+	CLSControlEditor(regionControl, parent)
 {
 
 }
```

Comment:
@iainworkman good catch. I made the changes yesterday last minute before I left and forgot to save the cpp file :).